src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.Collections;
+
 public class WorkWithFile {
+    static final int INDEX_ZERO = 0;
+    static final int INDEX_ONE = 1;
```

Comment:
you didn't fix the comment from the previous review. the goal of having a constant instead of a magic number is that a constant can have **meaningful** naming. How a person who reads this code should understand what these constants represent? 
choose more meaningful naming for the constans